From<String> missing for Robj?

It looks like a `From<String>` implementation is missing for `Robj`.

I tried to compile the following code:
```rust
use extendr_api::*;

#[extendr]
fn md_to_html(input: &str) -> String {
    markdown::to_html(input)
}
```
and got this compiler error:
```
error[E0277]: the trait bound `extendr_api::robj::Robj: std::convert::From<std::string::String>` is not satisfied
 --> src/lib.rs:3:1
  |
3 | #[extendr]
  | ^^^^^^^^^^ the trait `std::convert::From<std::string::String>` is not implemented for `extendr_api::robj::Robj`
  |
  = help: the following implementations were found:
            <extendr_api::robj::Robj as std::convert::From<&'a [&str]>>
            <extendr_api::robj::Robj as std::convert::From<&'a [extendr_api::logical::Bool]>>
            <extendr_api::robj::Robj as std::convert::From<&'a [i32]>>
            <extendr_api::robj::Robj as std::convert::From<&'a str>>
          and 22 others
  = note: required by `std::convert::From::from`
  = note: this error originates in an attribute macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to previous error
```

Alternatively, the following code compiles:
```rust

use extendr_api::*;

#[extendr]
fn md_to_html(input: &str) -> Robj {
    Robj::from(&*markdown::to_html(input))
}
```

--- a/extendr-api/src/robj.rs
+++ b/extendr-api/src/robj.rs
@@ -1351,6 +1351,14 @@ impl<'a> From<&'a str> for Robj {
     }
 }
 
+/// Convert a String to an Robj string array object.
+impl From<String> for Robj {
+    fn from(val: String) -> Self {
+        Robj::from(&*val)
+    }
+}
+
+/// Convert an array of string refs to an Robj string array object.
 impl<'a> From<&'a [&str]> for Robj {
     fn from(vals: &'a [&str]) -> Self {
         unsafe {
@@ -1565,6 +1573,8 @@ mod tests {
         assert_eq!(format!("{:?}", Robj::from(1)), "1");
         assert_eq!(format!("{:?}", Robj::from(1.)), "1.0");
         assert_eq!(format!("{:?}", Robj::from("hello")), "[\"hello\"]");
+        let s = "hello".to_string();
+        assert_eq!(format!("{:?}", Robj::from(s)), "[\"hello\"]");
 
         // Vectors
         assert_eq!(format!("{:?}", Robj::from(&[1, 2, 3][..])), "[1, 2, 3]");

